CONFIGURING_CLION.md

```diff
@@ -0,0 +1,24 @@
+# Configuring Clion for Rust-only development
+These are the steps to build the application-services desktop in clion:
+This works on Ubuntu 18.04 and OSx10.0, but will probably work on anything.
+
+First you will need to install the IntelliJ Rust plugin. https://plugins.jetbrains.com/plugin/8182-rust?_ga=2.3358832.1169745618.1582237573-208076843.1581265366
+
+When opening the project, Clion will probably automatically try to run the .gradle file. By creating our own build configuration in Clion we can avoid this get a more flexibal work environment. To build the project with Clion we can to do the following:
+
+### Create a new build configuration.
```

Comment:
Reorder this with below. Write something like:

"Update the Cargo Command template with required environment variables..."

and then below:

"Now create a build configuration using the Cargo Command template. For example, by using "`test --all`" as the command."